sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_service_groundedness/_service_groundedness.py

```diff
@@ -105,18 +105,18 @@ def __init__(
     def __call__(
         self,
         *,
+        response: str,
+        context: str,
         query: Optional[str] = None,
```

Comment:
```suggestion
        query: str,
```

Reply:
the service based evaluator currently requires query -- we want to make it optional in the future but for now it should be required.